src/command-bus.ts

```diff
@@ -49,6 +49,11 @@ export class CommandBus<CommandBase extends ICommand = ICommand>
     this._publisher = _publisher;
   }
 
+  private getCommandName(command: Type<CommandBase>): string {
+    const { constructor } = Object.getPrototypeOf(command);
+    return constructor.name as string;
+  }
```

Comment:
nit: one tiny question - could we move private methods below protected ones?

Reply:
done!